### ISSUE TITLE
http://zdc.stfx.eu redirects to https://audio359.eu/ 

### ISSUE THREAD

**Issue by maenu** [svenvc/zodiac#5]
Is the redirect intentional, or did some DNS server mess up something (mine or yours)?

**Comment by svenvc** [comment]
yes, that is an error, but the site there is very old and not very informative, all my code lives on github now.
